### PR TITLE
Completed javascript-view-swapping

### PR DESCRIPTION
**Pull request by Aiden-Peace300**
image
image
image

### CODE REVIEW

File: javascript-view-swapping/main.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// This line of code selects the first HTML element with the class "tab-container" and assigns it to the variable "$tabContainer" using JavaScript.
+const $tabContainer = document.querySelector('.tab-container');
+
+// selects the first HTML element with the class "tab-container" and assigns it to the variable "$tabContainer" using JavaScript.
+const $tabChildNodes = document.querySelectorAll('.tab');
+
+// selects the All the HTML element with the class ".view" and assigns it to the variable "$ViewChildNodes" using JavaScript.
+const $ViewChildNodes = document.querySelectorAll('.view');
```
Comment (shawnkost) [inline javascript-view-swapping/main.js:8]: Variable names should always start with lowercase

File: javascript-view-swapping/main.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  // iterating though ViewChildNodes ("array")
+  for (let i = 0; i < $ViewChildNodes.length; i++) {
+    // view will hold text content for the tab clicked
+    const view = $ViewChildNodes[i];
+
+    // if current attribute of 'data-view' value in the view object is the same as dataViewAttrValue ...
+    if (view.getAttribute('data-view') === $dataViewAttrValue) {
+      // ... execute the following
+      // adding 'view' class to the current view
+      console.log('IF $dataViewAttrValue: ', $dataViewAttrValue);
```
Comment (shawnkost) [inline javascript-view-swapping/main.js:46]: Remove all console logs before submitting! Unless we specifically ask for you to include them

File: javascript-view-swapping/main.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      view.classList.add('view');
+      view.classList.remove('hidden');
+    } else {
+      // if current view is NOT the same as dataViewAttrValue ...
+      // remove 'hidden' class to the current view
+      console.log('ELSE $dataViewAttrValue: ', $dataViewAttrValue);
```
Comment (shawnkost) [inline javascript-view-swapping/main.js:52]: Remove all console logs before submitting! Unless we specifically ask for you to include them

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just a few small tweaks here Aiden!

File: javascript-view-swapping/main.js (hunk 3 above)
Comment (Aiden-Peace300) [reply javascript-view-swapping/main.js]: My apologies, Slipped my mind!

File: javascript-view-swapping/main.js (hunk 2 above)
Comment (Aiden-Peace300) [reply javascript-view-swapping/main.js]: My apologies, Slipped my mind!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Comment by Aiden-Peace300** [comment]
Corrections made

**Pull request merged**